### PR TITLE
Extend documentation; configure server-side CSRF checks for sample

### PR DESCRIPTION
**Pull request by whummer**
* extend README documentation on how to start and interact with the demo app
* set `DISABLE_CORS_CHECKS=1` in Helm deployment to disable server-side CSRF checks

### CODE REVIEW

File: l8k/install.py [hunk 1]
```diff
@@ -64,6 +65,14 @@ def install(self):
 
     def create(self):
         bin_file = self.install()
+
+        # check if cluster exists
+        clusters = json.loads(run([bin_file, "cluster", "list", "-o", "json"]))
+        matching = [c for c in clusters if c["name"] == K3D_CLUSTER_NAME]
+        if matching:
+            LOG.debug("Target k8d cluster '%s' already exists", K3D_CLUSTER_NAME)
```
Comment (dominikschubert) [inline l8k/install.py]: `k8d`?

File: README.md [hunk 2]
```diff
@@ -24,14 +25,60 @@ To create an embedded Kubernetes (k3d) cluster in Docker and install LocalStack
...
+    "items": [
+        {
+            "id": "ses2pi5oap",
+            "name": "local-localstack-demo",
+...
+```
+
+We can then use a browser to open the Web UI, which should have been deployed to an S3 bucket inside LocalStack. The Web UI can be used to interact with the sample application, send new requests to the backend, inspect the state of existing requests, etc.
+```
+$ open http://localhost:8081/archive-bucket/index.html
```
Comment (dominikschubert) [inline README.md]: this is only targeting macos. Probably better to just add a link here.

```suggestion
We can then use a browser to open the [Web UI](http://localhost:8081/archive-bucket/index.html), which should have been deployed to an S3 bucket inside LocalStack. The Web UI can be used to interact with the sample application, send new requests to the backend, inspect the state of existing requests, etc.
```

**Review by dominikschubert — APPROVED** [review]
small nits, otherwise lgtm

File: l8k/install.py (hunk 1 above)
Comment (whummer) [reply l8k/install.py]:
```suggestion
            LOG.debug("Target k3d cluster '%s' already exists", K3D_CLUSTER_NAME)
```

**Pull request merged**